testsuite/tests/typing-unboxed/test.ml.reference

```diff
@@ -1,5 +1,25 @@
 
-#                                                           external a : (int [@untagged]) -> unit = "a"
+#                                                           Characters 816-860:
+Warning 60: Ignored attribute in external declaration: no native code version of the primitive provided
```

Comment:
Is it intentional that the testsuite would raise those warning here? Shouldn't the testsuite be fixed?